### PR TITLE
[DOC] Updates to conform to `ruff` - `pydocstyle` hook

### PR DESCRIPTION
**Pull request by AnonymousCodes911**
Adds missing docstrings and ensures they comply with the `ruff` linter.

#### Reference Issues/PRs
fixes: #1191,  from: #1055 
part of : #1196

### CODE REVIEW

File: aeon/base/tests/test_base_series.py [hunk 1]
```diff
@@ -78,6 +78,7 @@ def test__check_X():
 
 
 def test_univariate_convert_X():
+    """Test to check Univariate."""
```
Comment (MatthewMiddlehurst) [inline aeon/base/tests/test_base_series.py:81]: Be more descriptive. i.e. `"""Test _covert_X on univariate series."""`

File: aeon/base/tests/test_base_series.py [hunk 2]
```diff
@@ -146,10 +149,34 @@ def test__preprocess_series(input_type, inner_type):
...
+    Test the _check_y method with correct input.
+
+    Parameters
+    ----------
+    y_correct : object
+        A correct input object for the _check_y method.
+
+    Returns
+    -------
+    None
```
Comment (MatthewMiddlehurst) [inline aeon/base/tests/test_base_series.py:162]: I don't think these bits are necessary to be honest. Just keep them simple for now.

File: aeon/base/estimator/hybrid/tests/test_base_rist.py [hunk 3]
```diff
@@ -75,6 +79,7 @@ def test_rist_estimator_input():
     reason="skip test if required soft dependency not available",
 )
 def test_rist_series_transform_input():
+    """Test the RISTClassifier class with Tansform input."""
```
Comment (MatthewMiddlehurst) [inline aeon/base/estimator/hybrid/tests/test_base_rist.py:82]: Spelling mistake here, more generally be careful with capitalisation where its not necessary.

**Review by MatthewMiddlehurst — COMMENTED** [review]
If this is in progress it may be better as a draft PR. Don't close the issue unless it has all been solved.

File: aeon/base/estimator/hybrid/tests/test_base_rist.py (hunk 3 above)
Comment (AnonymousCodes911) [reply aeon/base/estimator/hybrid/tests/test_base_rist.py:82]: Got it!

File: aeon/base/tests/test_base_series.py (hunk 2 above)
Comment (AnonymousCodes911) [reply aeon/base/tests/test_base_series.py:162]: Will keep this in mind!

File: aeon/base/tests/test_base_series.py (hunk 1 above)
Comment (AnonymousCodes911) [reply aeon/base/tests/test_base_series.py:81]: Understood!

**Comment by AnonymousCodes911** [comment]
Certainly, feel free to proceed as you see fit!
Thank you!

**Comment by MatthewMiddlehurst** [comment]
@all-contributors add @AnonymousCodes911 for docs

**Review by MatthewMiddlehurst — APPROVED** [review]

**Pull request merged**